src/summary/summary.js

```diff
@@ -0,0 +1,32 @@
+export class SummaryComponent {
+  #water = document.getElementById("water-event-count");
+  #stretch = document.getElementById("stretch-event-count");
+  #break = document.getElementById("break-event-count");
+  #task = document.getElementById("tasks-done-count");
+  constructor() {
+  }
+
+  displayCount(type, number) {
+    if (!type) {
+      return
+    };
+    switch(type) {
+      case "Water":
+        this.#water.innerText = number;
+        break;
+      case "Stretch":
+        this.#stretch.innerText = number;
+        break;
+      case "Break":
+        this.#break.innerText = number;
+        break;
+      case "Tasks":
+        this.#task.innerText = number;
+      default:
+        break;
+    };
```

Comment:
You can refactor this to be something like:

```js
if (this.#doms[type].innerText) {
  this.#doms[type].innerText = number
}
```
Which means move all the #task, #break etc into:
```js
#doms = {
  Water: document.getElementById("water-event-count"),
  ...
}
```

Then you do not need so much if-else